modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/AutomationComponent.java

```diff
@@ -135,21 +136,23 @@ public void registerContribution(Object contribution, String extensionPoint, Com
                     widgetDefinitionList.add(widgetDescriptor.getWidgetDefinition());
                 }
             }
+            Class<?> type;
```

Comment:
Not sure you need this declaration outside the try/catch?

Reply:
indeed